inscriptions/ISic001229.xml

```diff
@@ -190,20 +195,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140715">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#GC">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Ἀγριππεί-<lb n="2"/>νῳ μου-<lb n="3"/>σικῷ μν-<lb n="4"/> ημόσυ-<lb n="5"/> νον.
-                    <lb n="6"/>
+                    <lb n="1"/><name>Ἀγριππεί
+                    <lb n="2" break="no"/>νῳ</name> μου
+                    <lb n="3" break="no"/>σικῷ μν
+                    <lb n="4" break="no"/>ημόσυ
+                    <lb n="5" break="no"/>νον
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app><note>Text of IG</note></app>
+                </listApp>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#GC">
+                <p>To Agrippeinous, musical, as memorial.</p>
```

Comment:
mousikos could either be rendered literally as 'devoted to the Muses', or else assumed to mean either poet or musician by transfer. I would suggest to render as 'A memorial for Agrippeinos, devotee of the Muses'